src/middleware.js

```diff
@@ -54,11 +55,11 @@ function morganMiddleware(req, res, next) {
     });
 
     const logger = getContainer('logger');
-    const formatFormat = json(format);
     const options = {
         stream: asStream(logger),
         skip: skip(ignoreRouteUrls),
     };
+    const formatFormat = compile(format, options);
```

Comment:
Those options aren't implemented and the object appears to be for `morgan` on line 63